.github/workflows/build.yml

```diff
@@ -102,11 +102,20 @@ jobs:
       - name: Pre-npm-Publish
         run: |
           yarn --skip-integrity-check --network-timeout 100000
-          yarn compile
```

Comment:
not needed?

Reply:
Not needed: This script is part of of `prepare` being run after `yarn install` ✔️